fmt/sbom_fmt.yml

```diff
@@ -1,7 +1,7 @@
 name: fmt
-version: 11.0.1
+version: 11.1.3
 cpe: cpe:2.3:a:fmt:fmt:{}:*:*:*:*:*:*:*
-supplier: 'Organization: fmt <https://fmt.dev/latest/index.html>'
+supplier: "Organization: fmt <https://fmt.dev/latest/index.html>"
 description: A modern formatting library
 url: https://github.com/fmtlib/fmt/
 hash: b50e685db996c167e6c831dcef582aba6e14276a
```

Comment:
@euripedesrocha I think this hash has to be updated to the new commit of the submodule.